front/app/components/IdeaCards/IdeasList.tsx

```diff
@@ -74,6 +67,9 @@ interface Props {
   participationContextType?: IParticipationContextType | null;
   ariaLabelledBy?: string;
   tabIndex?: number;
+  hideImage: boolean;
+  hideImagePlaceholder: boolean;
+  hideIdeaStatus: boolean;
```

Comment:
Making this reusable in the IdeaCards component with or without filters.

Reply:
Shouldn't these props be optional?